src/evt.js

```diff
@@ -36,11 +28,29 @@ evt.prototype.raise = function() {
 };
 
 function evtInit(element) {
-
-  // handle the element/selector
-
-  return new evt();
+  if (!element || (element instanceof Element) === false) {
+    throw new Error('You must supply a DOM element or selector to evt');
+  }
+
```

Comment:
Apologies, yeah. Just needs the string check. The whole collection thing is a separate issue...